ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
```

Comment:
Yeah, I had it originally split out for the pulsing effect if we wanted to extend it to the pin button in the action bar or something. Specifically doing this for just retrigger will simplify this a lot.